app/controllers/middleware_deployment_controller.rb

```diff
@@ -27,52 +26,18 @@ class MiddlewareDeploymentController < ApplicationController
     }
   }.freeze
 
-  def show
-    clear_topology_breadcrumb
-    @display = params[:display] || "main" unless control_selected?
-    @lastaction = "show"
-    @showtype = "main"
-    @record = identify_record(params[:id])
-    show_container(@record, controller_name, display_name)
-  end
-
   def button
     selected_operation = params[:pressed].to_sym
     if OPERATIONS.key?(selected_operation)
-      selected_archives = identify_selected_deployments
-      run_deployment_operation(OPERATIONS.fetch(selected_operation), selected_archives)
+      selected_archives = identify_selected_entities
```

Comment:
I didn't choose or modified the name of the variable, but I assume that the name is because you select jar archives to deploy/undeploy, probably...